features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/AppsAndKeys/TokenManager.jsx

```diff
@@ -330,6 +375,31 @@ class TokenManager extends React.Component {
         if (!keys) {
             return <Loading />;
         }
+        const keysOfKeyType = keys.get(keyType);
```

Comment:
Are the keysOfKeyType and key variable below duplicated?

Reply:
Fixed.